src/App.css

```diff
@@ -250,16 +287,76 @@ nav {
   top: 0px;
   content: '';
   margin-right: 10px;
-  height: 32px;
-  width: 32px;
+  height: 4.2ch;
+  width: 4.2ch;
   border: 2px solid rgb(33, 33, 33);
 }
 
-.prompt {
-  display: grid;
-  grid-template-rows: auto auto;
-  grid-template-columns: auto;
-  grid-row-gap: 20px;
-  text-align: center;
+.list__item__delete-btn {
+  display: inline-flex;
+  align-items: center;
   justify-content: center;
+  flex-shrink: 0;
+  width: 4.5ch;
+  height: 4.5ch;
+  border-radius: none;
+  background: none;
+  border: none;
+  fill: var(--theme-color);
```

Comment:
I would suggest you add a `cursor: pointer;` here to make it more obvious that the trash can is a button :slightly_smiling_face:

Reply:
Oh yes, you're right. Will add this